client/src/qa/components/QuestionAndAnswerEntry.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import QuestionEntry from './QuestionEntry';
+import AnswerEntry from './AnswerEntry';
+
+const QuestionAndAnswerEntry = ({ question, answers }) => {
+  const [slice, setSlice] = useState(2);
```

Comment:
I noticed that you have setSlice as your second argument. Will you be using it to change slice so that on a button click. const [slice, setSlice] = useState(2); --> const [slice, setSlice] = useState(4);

Reply:
Yep!